src/classes/STG_PanelADDRVerification_CTRL.cls

```diff
@@ -35,10 +35,47 @@
 */
 public with sharing class STG_PanelADDRVerification_CTRL extends STG_Panel {
 
+    /*******************************************************************************************************
+    * @description Interface for the address verifier
+    */
+    public Interface IAddressVerifier {
+        String verifyService(Addr_Verification_Settings__c settings, String zipCode);
+        Boolean hasAccess();
+        void encodeToken(Addr_Verification_Settings__c settings);
+    }
+
+    private static final String SMARTY_STREETS_VALIDATOR_CLASS = 'ADDR_SmartyStreets_Validator';
```

Comment:
Should this use `ADDR_SmartyStreets_Validator.class.name()` rather than hardcoding?

Reply:
Good point, it has been done.

Reply:
The `class.getName()` is removed since the variable should not include the namespace, so I changed it back to be hardcoded value as before. Let me know if there are issues with the approach.

Reply:
The change has been done, I left the hardcoded name in the test class, since we would like to verify the value as expected is used in the controller.